src/i18n-module-wordpressorg.php

```diff
@@ -26,8 +26,9 @@ public function __construct( $args, $show_translation_box = true ) {
 	}
 
 	/**
-	 * Returns the i18n_promo message from the i18n_module. Returns en empty string if the
-	 * promo shouldn't be shown.
+	 * Returns the i18n_promo message from the i18n_module if the i18n promo should be shown.
```

Comment:
It seems useful information to document an empty string will be returned if the promo message shouldn't be shown. Why was it removed?

Reply:
not sure why I did that. Reverting this.